components/Modal/ModalShare.tsx

```diff
@@ -0,0 +1,82 @@
+import { modal } from "../../types/modal.type";
+import closeBtn from "../../public/icons/close.svg";
+import kakaoImg from "../../public/icons/kakao.svg";
+import facebookImg from "../../public/icons/facebook.svg";
+import linkImg from "../../public/icons/link.svg";
+import Image from "next/image";
+import styles from "./ModalShare.module.css";
+import Link from "next/link";
+
+const ModalShare = ({ closeModal, modalTitle }: modal) => {
+  const hostAddress = "https://yourdomain.com"; // 각자 주소 바꾸기
+  const shareUrl = `${hostAddress}/shared/${1}`; // id값도 바꾸기
```

Comment:
각 주석의 의미에 맞게 컨벤션을 정해 작성하면, 추후 확인 및 유지보수 시 좋습니다!
일반적으로, `// TODO:`, `// TOFIX:`와 같이 사용합니다.

```suggestion
  const hostAddress = "https://yourdomain.com"; // TODO: 각자 주소 바꾸기
  const shareUrl = `${hostAddress}/shared/${1}`; // TODO: id값도 바꾸기
```